platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,462 @@
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://1000genomes`
+    - A unique **Name** for the bucket, such as `1000G`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own data for interactive analysis, see [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for instructions to add your own public or private cloud bucket. 
+:::
+
+#### Create an Xpra studio 
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- In the **Compute & Data** tab:
+    - Select your AWS Batch compute environment. 
+      :::note
+      Studios compete for computing resources when sharing compute environments. Ensure your compute environment has sufficient resources to run both your pipelines and data studio sessions. The default CPU and memory allocation for a studio is 2 CPUs and 8192 MB RAM. 
+      :::
+    - Optional: Enter CPU and memory allocations.  
+    - Mount the 1000 Genomes S3 bucket you added previously using Data Explorer. 
+- In the **General config** tab:
+    - Select the latest **Xpra** container image template from the list.
+    - Optional: Enter a unique name and description for the studio. 
+    - Check **Install Conda packages** and paste the following into the YML field:
+
+        ```yaml 
+        channels:
+          - conda-forge
+          - bioconda
+        dependencies:
+          - igv
+          - samtools
+        ```
+
+- Select **Add** or choose to **Add and start** the studio immediately.
+- If you chose to **Add** the studio in the preceding step, select **Connect** in the options menu to open the studio in a new browser tab. 
+
+#### View variants in IGV desktop
+
+1. In the Xpra terminal, run `igv` to open IGV desktop.
+1. In IGV, change the genome version to hg19.
+1. Select **File**, then **Load from file**, then navigate to `/workspace/data/xpra-1000Genomes/phase3/data/HG00096/high_coverage_alignment` and select the `.bai` file, as shown below:
+
+    ![Load BAM file in IGV desktop](./_images/xpra-data-studios-IGV-load-bam.png)
+
+1. Search for PCSK9 and zoom into one of the exons of the gene. A coverage graph and reads should be shown, as below:
+
+    ![BAM file view](./_images/xpra-data-studios-IGV-view-bam.png)
+
+### VS Code: Create an interactive Nextflow developer environment
+
+Using Studios with VS Code allows you to set up a portable and interactive Nextflow developer environment that contains all the tools you need to code and run Nextflow pipelines. This section will demonstrate how to set up a VS Code Studio with Conda, nf-core tools, and Apptainer, add public data and run the nf-core/fetchngs pipeline with the `test` profile, and create a VS Code project to start coding your own Nextflow pipelines.  
```

Comment:
```suggestion
Using Studios and Visual Studio Code allows you to create a portable and interactive Nextflow development environment with all the tools you need to develop and run Nextflow pipelines. This section demonstrates how to set up a VS Code Studio with Conda, nf-core tools, and Apptainer, add public data and run the nf-core/fetchngs pipeline with the `test` profile, and create a VS Code project to start coding your own Nextflow pipelines.  
```

Reply:
I would completely remove apptainer from this version, as currently it doesn't work inside VSCode studios due to the way we are launching and running studios. Let's focus on conda as the environment engine to use when developing inside VSCode studios for now!